x-pack/platform/plugins/private/upgrade_assistant/server/routes/status.ts

```diff
@@ -99,7 +124,7 @@ export function registerUpgradeStatusRoute({
               })
             );
           }
-
+          // can be improved by showing health indicator issues separately
```

Comment:
I was tempted to add a new upgrade issue type for health indicators but that might be confusing, since all the issues were grouped together and indicated as such.